nodes.go

```diff
@@ -296,23 +296,19 @@ type renameNode struct {
 }
 
 // Rename node
-func (n *Node) Rename(newName string) (*Node, *http.Response, error) {
```

Comment:
I'd prefer to return the Node that Amazon Drive returns - so I don't think we need to change this, unless you know something I don't?